lib/token-assert.js

```diff
@@ -77,11 +77,21 @@ TokenAssert.prototype.noWhitespaceBetween = function(options) {
  * @param {Object} options.token
  * @param {Object} options.nextToken
  * @param {String} [options.message]
+ * @param {Boolean} [options.requireNextToken=false]
  */
 TokenAssert.prototype.sameLine = function(options) {
     var token = options.token;
     var nextToken = options.nextToken;
 
+    if (options.requireNextToken && (!nextToken || nextToken.type === 'EOF')) {
+        this.emit('error', {
+            message: options.message || token.value + ' should not be the last in the file',
```

Comment:
It seems weird to be messaging about the end of the file in a generic helper like this.